fdbclient/NativeAPI.actor.cpp

```diff
@@ -2411,13 +2417,16 @@ TransactionOptions::TransactionOptions(Database const& cx) {
 TransactionOptions::TransactionOptions() {
 	memset(this, 0, sizeof(*this));
 	maxBackoff = CLIENT_KNOBS->DEFAULT_MAX_BACKOFF;
+	customTransactionSizeLimit = CLIENT_KNOBS->TRANSACTION_SIZE_LIMIT;
 }
 
 void TransactionOptions::reset(Database const& cx) {
 	double oldMaxBackoff = maxBackoff;
 	double oldMaxRetries = maxRetries;
+	uint32_t oldSizeLimit = customTransactionSizeLimit;
 	memset(this, 0, sizeof(*this));
 	maxBackoff = cx->apiVersionAtLeast(610) ? oldMaxBackoff : cx->transactionMaxBackoff;
+	customTransactionSizeLimit = oldSizeLimit;
```

Comment:
This is to make the custom size limit persist across reset (like max backoff, max retries, and timeout)? I think that's probably fine (and makes sense for consistency with those options, I suppose), though we should document that.

Reply:
Added to 'option-set-size-limit-blurb' and 'option-db-tr-size-limit-blurb' in the documentation.